sale_cancel_reason/model/sale.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    cancel_reason_id = fields.Many2one(
+        "sale.order.cancel.reason",
```

Comment:
Shouldn't be used attribute names for all field's attributes? If yes there are few other places with the same issue.